recipes/xnvme-sys/meta.yaml

```diff
@@ -0,0 +1,74 @@
+---
+{% set name = "xnvme" %}
+{% set version = "0.7.4" %}
+{% set sha256 = "6d42d0bd7e6b395a37869a8713d138d332500f20ecc272ebf6789471026a7191" %}
+
+package:
+  name: {{ name|lower }}-sys
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [not linux]
+  script: |
+    meson setup builddir --prefix=$PREFIX -Ddefault_library=shared -Dwith-spdk=false 
+    meson compile -C builddir
+    meson install -C builddir
+
+    # Create activation and deactivation scripts
+    mkdir -p $PREFIX/etc/conda/activate.d
+    mkdir -p $PREFIX/etc/conda/deactivate.d
+
+    # Activation script
+    echo 'export PATH=$PREFIX/bin:$PREFIX/usr/bin:$PREFIX/usr/local/bin:$PREFIX/usr/local/lib/x86_64-linux-gnu:$PATH' > $PREFIX/etc/conda/activate.d/xnvme.sh
```

Comment:
Please ensure that the binaries are installed in typical conda paths. You should not need to adjust PATH with activation scripts.

Reply:
This is giving me some challenges when using ``pkg-config``. From what I can tell then the ``build.sh`` does as other meson-projects do:

```
meson setup builddir --prefix=$PREFIX
```

However, the command in ``run_test.sh`` which checks whether ``pkg-config`` works, it fails unless these things are setup in the activation / de-activation scripts. It would be great with a pointer on how this should otherwise be done.

Reply:
You probably need to pass `$MESON_ARGS` to meson to get them right.

Reply:
Thanks @xhochy ! This is the thing that was missing.